src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/IllegalTypeCheck.java

```diff
@@ -130,13 +130,37 @@
  * </li>
  * </ul>
  * <p>
- * To configure the check so that it ignores getInstance() methods:
+ * Default Configuration:
+ * </p>
+ * <pre>
+ * &lt;module name=&quot;IllegalType&quot;/&gt;
+ * </pre>
+ * <pre>
+ * public class Test {
+ *   public void function() {
+ *     HashMap&lt;String, String&gt; var; // violation
```

Comment:
please do not use `var` as variable name, it can be a bit confusing after java 10

Reply:
done